curator/templates/single.html

```diff
@@ -1,28 +1,121 @@
+
+<!-- wp:cover {"useFeaturedImage":true,"dimRatio":50,"overlayColor":"foreground","minHeight":100,"minHeightUnit":"vh"} -->
+<div class="wp-block-cover" style="min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-foreground-background-color has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"center"}} -->
+<div class="wp-block-group"><!-- wp:post-terms {"term":"post_tag","textColor":"background","fontSize":"medium"} /-->
+
+<!-- wp:post-date {"textColor":"background","fontSize":"medium"} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:post-title {"textAlign":"center","level":1} /--></div></div>
+<!-- /wp:cover -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:columns {"verticalAlignment":"center","align":"wide"} -->
+<div class="wp-block-columns alignwide are-vertically-aligned-center"><!-- wp:column {"verticalAlignment":"center","width":"100%","layout":{"inherit":false}} -->
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:100%"><!-- wp:post-excerpt {"textAlign":"center","fontSize":"x-large"} /--></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:post-content {"layout":{"inherit":true}} /--></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px","bottom":"30px"}},"border":{"top":{"color":"var:preset|color|foreground","width":"2px"},"bottom":{"color":"var:preset|color|foreground","width":"2px"}}},"layout":{"inherit":false}} -->
+<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--foreground);border-top-width:2px;border-bottom-color:var(--wp--preset--color--foreground);border-bottom-width:2px;padding-top:30px;padding-bottom:30px"><!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group"><!-- wp:post-navigation-link {"type":"previous","label":"\u003cstrong\u003ePREVIOUS POST\u003c/strong\u003e","fontSize":"large"} /-->
+
+<!-- wp:post-navigation-link {"label":"\u003cstrong\u003eNEXT POST\u003c/strong\u003e","fontSize":"large"} /--></div>
```

Comment:
The font family should be Work Sans for the pagination: 

<img width="796" alt="Screen Shot 2022-05-10 at 11 30 30 AM" src="https://user-images.githubusercontent.com/5375500/167666222-278a247e-75c3-4be2-9df2-758142aa092b.png">